src/api/dependencies.py

```
@@ -58,44 +58,18 @@ def get_account_from_bearer(
 
     if user is None:
         raise credentials_exception
-    
-    if not user.is_active:
-        raise HTTPException(status_code=400, detail="Inactive account")
 
     return user
 
 def get_active_account(account: Account = Depends(get_account_from_bearer)) -> Account:
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)